assets/scripts/script.js

```diff
@@ -58,7 +58,8 @@
         thumbnail: true,
         showThumbByDefault: false,
         subHtmlSelectorRelative: true,
-        selector: '.woocommerce-product-gallery__image > a'
+        selector: '.woocommerce-product-gallery__image > a',
+        width: '1010px'
```

Comment:
@evemf — As we discusses in person, please ensure that this is working properly as well with images smaller than 1010px. We do not want stretched or blurred images. :)